src/tags/visual/View.js

```diff
@@ -110,8 +112,15 @@ const HtxView = observer(({ item }) => {
     style["display"] = "none";
   }
 
+  const className = [
+    item.className,
+    styles.viewContainer,
+  ]
+    .filter(Boolean)
+    .join(" ");
+
   return (
-    <div className={item.classname} style={style}>
+    <div className={isFF(FF_DEV_2395) ? className : item.className} style={style}>
```

Comment:
With that you can just leave the `className={item.className}`

Reply:
done